src/admin/content/helpers/content-types/content.ts

```diff
@@ -0,0 +1,34 @@
+import { Avo } from '@viaa/avo2-types';
+
+import { fetchContentItems } from '../../content.services';
+
+// Fetch content items from GQL
+export const fetchContent = (limit: number = 5) => {
+	return fetchContentItems(limit).then((contentItems: Avo.Content.Content[] | null) => {
```

Comment:
use async await instead of promises:
```typescript
// Fetch content items from GQL
export const fetchContent = async (limit: number = 5): Promise<PickerSelectItemGroup> => {
	const contentItems: Avo.Content.Content[] | null = await fetchContentItems(limit);
	return parseContentItems(contentItems || []);
};

// Parse raw content items to react-select options
const parseContentItems = (raw: Avo.Content.Content[]): PickerSelectItemGroup => {
	const parsedContentItems: PickerSelectItem[] = raw.map((item: Avo.Content.Content) => ({
		label: item.title,
		value: {
			type: 'content',
			value: item.path,
		},
	}));

	return {
		label: 'content',
		options: parsedContentItems,
	};
};
```